.gitattributes

```diff
@@ -0,0 +1,16 @@
+# Set the default behavior, in case people don't have core.autocrlf set.
+* text=auto
+
+# Explicitly declare text files you want to always be normalized and converted
+# to native line endings on checkout.
+*.c text
+*.h text
+*.py text eol=lf
+
+# Declare files that will always have CRLF line endings on checkout.
+*.sln text eol=crlf
+
+# Denote all files that are truly binary and should not be modified.
+*.png binary
+*.jpg binary
```

Comment:
Is this really necessary for .png/.jpg? Asking cause this seems a bit unnecessary in the sense that these are usually files that don't cause trouble for git to define them as binary. I hope I'm not being too nitpicky, it is just that removing the unnecessary often improves the quality by removing noise/unnecessary info.